tools/jenkins-run.sh

```diff
@@ -43,7 +45,7 @@ if [[ ${N_TOTAL_CONTAINER} -ne 0 ]]; then
     exit
 fi
 
-DOCKER_OPT=()
+DOCKER_OPT=("-v" "/var/run/docker.sock:/var/run/docker.sock")
```

Comment:
Why do we need this?

Reply:
We need this to run docker from our container